contact/async.js

```diff
@@ -18,11 +19,7 @@ exports.create = function (api) {
   })
 
   function followerOf (source, dest, cb) {
-    var following = api.contact.obs.following(source)
-    onceTrue(following.sync, () => {
-      var value = resolve(following)
-      cb(null, value && value.has(dest))
-    })
+    api.sbot.async.friendsGet({source: source, dest: dest}, cb)
```

Comment:
yes. if value isn't passed in `undefined && foo` will return `undefined` && and || etc operators only evaluate the second argument if it needs to.